cdc/owner/changefeed.go

```diff
@@ -321,10 +322,16 @@ func (c *changefeed) releaseResources(ctx cdcContext.Context) {
 	c.redoManagerCleanup(ctx)
 	canceledCtx, cancel := context.WithCancel(context.Background())
 	cancel()
+
+	start := time.Now()
 	// We don't need to wait sink Close, pass a canceled context is ok
-	if err := c.sink.close(canceledCtx); err != nil {
-		log.Warn("Closing sink failed in Owner", zap.String("changefeed", c.state.ID), zap.Error(err))
+	err := c.sink.close(canceledCtx)
+	if err != nil {
+		log.Warn("Closing ddl sink failed in Owner", zap.String("changefeedID", c.state.ID), zap.Error(err), zap.Duration("elapsed", time.Since(start)))
```

Comment:
```suggestion
		log.Warn("Closing ddl sink failed in Owner", zap.String("changefeedID", c.state.ID), zap.Error(err), zap.Duration("duration", time.Since(start)))
```